src/amo/components/HomepageShelves/index.js

```diff
@@ -29,6 +31,8 @@ type InternalProps = {|
 export const HOMESHELVES_ENDPOINT_COLLECTIONS = 'collections';
 export const HOMESHELVES_ENDPOINT_SEARCH = 'search';
 export const HOMESHELVES_ENDPOINT_SEARCH_THEMES = 'search-themes';
+export const HOMESHELVES_ADDON_TYPE_EXTENSIONS = ADDON_TYPE_EXTENSION;
```

Comment:
I don't think we need these `const`s. We can just use `ADDON_TYPE_EXTENSION` and `ADDON_TYPE_STATIC_THEME` directly in the code. We do that everywhere else.